packages/block-directory/src/store/test/controls.js

```diff
@@ -1,45 +1,21 @@
 /**
  * Internal dependencies
  */
-import { loadScript, loadStyle } from '../controls';
+import { loadAsset } from '../controls';
 
 describe( 'controls', () => {
-	const scriptAsset = 'http://www.wordpress.org/plugins/fakeasset.js';
-	const styleAsset = 'http://www.wordpress.org/plugins/fakeasset.css';
+	describe( 'loadAsset', () => {
+		const script = document.createElement( 'script' );
+		const style = document.createElement( 'link' );
 
-	describe( 'loadScript', () => {
 		it( 'should return a Promise when loading a script', () => {
-			const result = loadScript( scriptAsset );
+			const result = loadAsset( script );
 			expect( typeof result.then ).toBe( 'function' );
 		} );
 
-		it( 'should reject when no script is given', async () => {
-			expect.assertions( 1 );
-			const result = loadScript( '' );
-			await expect( result ).rejects.toThrow( Error );
-		} );
-
-		it( 'should reject when a non-js file is given', async () => {
```

Comment:
Correct. We don't actually care that much, as the new function just clones a supplied element and assumes that it's being passed valid data.
It doesn't necessarily have to be an external file, it could be an inline `<script>...</script>` or  an externally hosted JS/CSS file that doesn't use `.js` or `.css` extensions.